docs/workspaces/programming/python-workbench.qmd

```diff
@@ -101,11 +93,25 @@ To start using `miniconda` environments, activate `conda` with the command `cond
 
 Also see the [poetry docs](https://python-poetry.org/docs/).
 
-
 {{< include ../partials/_data_transfer.qmd >}}
 
 The recommended [iBridges client for Yoda and iRODS](../../manuals/ibridges.qmd) is preinstalled.
 
+#### VSCodium code editor (Desktop only)
+
+Under 'Applications' (topleft corner) you can find the code editor ['VSCodium'](https://vscodium.com/) (an open source alternative to VS Code) under 'Development'. VSCodium can be used for writing python scripts, Jupyter notebooks, etc.
+
+#### Jupyter lab interface (Desktop only)
```

Comment:
...or install the Jupyter extension for VSCodium?

Reply:
I clarified a bit that VSCodium is also a possibility